pom.xml

```diff
@@ -22,6 +22,11 @@
             <artifactId>jersey-client</artifactId>
             <version>2.25</version>
         </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
```

Comment:
Hi Guys, JUnit 5 is up and running. No sense to use 4th, http://junit.org/junit5/.